Ascend.java

```diff
@@ -3,5 +3,24 @@
 public class Ascend {
 	public static void main(String[] args) {
 		// Replace this comment with your code
+		int n = Integer.parseInt(args[0]);

```

Comment:
Bad names, Bad spacing, Bad indentation
Also: 
could merge 2 min and max calculation:
int min1 = Math.min(Math.min(a, b), c);
int max = Math.max(Math.max(a, b), c);